app/models/mortgage_calculator/tax_calculator.rb

```diff
@@ -12,9 +12,9 @@ class TaxCalculator
     validates :price, presence: true, numericality: true
 
     SECOND_HOME_THRESHOLD = 40_000
-    SECOND_HOME_ADDITIONAL_TAX = 3
     STANDARD_BUYER_TYPE = 'isNextHome'.freeze
```

Comment:
Is not possible to use `STANDARD_BUYER_TYPE` (`isNextHome`) instead of a specific `SECOND_PROPERTY_BUYER`?

Reply:
I'm not sure I understand the question. `isNextHome` is not the same as `isSecondHome`. They are different buyer types and require different calculations.

Reply:
Perfect then, that answers my question 👍